components/Post/Post.tsx

```diff
@@ -22,7 +22,7 @@ export const Post = ({ post }: Props) => (
     <h3>{post.frontmatter.title}</h3>
```

Comment:
In my free time I will improve this view a little bit cause it almost don't have styles right now and looks kinda ugly xD
I guess the most important thing is seo improvement so we can leave it for now